docs/installation/redis.rst

```diff
@@ -0,0 +1,24 @@
+.. _installation_redis:
```

Comment:
I'd expand this documentation section a bit more to provide context.

- [ ] Add a note at the top for the indented audience (DevOps), similar to https://open-forms.readthedocs.io/en/latest/installation/security.html

- [ ] Provide an introduction with high-level context:
    * Redis is used by Open Forms as cache, message broker and result store
    * Redis primarily is an in-memory store which implies data loss is possible unless configuration is tweaked

- [ ] Provide a section "configuring redis as robust message broker"
    * Explain the default config and RDB snapshotting here - point out the risk of data loss with default config
    * Explain AOF mechanism as easiest workaround (and note that the window for data loss is now reduced to max 1s)
    * Mention that alternatives to AOF are available but out of scope for our documentation: HA clustering, replication, sentinel... -> refer to the redis documentation. You could also mention that an alternative broker such as RabbitMQ could be an option (and configured through `CELERY_BROKER_URL`), but is out of scope for our documentation. Refer to the celery docs for this.

- [ ] Provide a section "Deploying multiple redis instances"
    * Point out that we share a single redis instance for cache, message broker and result store because of convenience
    * Point out that it's possible the AOF/clustering configuration *may* impact cache/result store performance and vice-versa (cache/result store use can lead to many writes to the AOF file which you might not want)
    * Document the option to deploy a redis instance for each individual use case and that these can be configured through the envvars `CELERY_BROKER_URL`, `CACHE_<suffix>` and `CELERY_RESULT_BACKEND` (link to the env configuration reference) so you can use different redis configuration files per instance, tuned for the intended usage.